invenio.cfg

```diff
@@ -274,7 +274,7 @@ USERPROFILES_EXTEND_SECURITY_FORMS = True  # automatically use user's email addr
 
 # OAI-PMH
 # =======
-OAISERVER_ID_PREFIX = "cds-rdm.com"
+OAISERVER_ID_PREFIX = "repository.cern"
```

Comment:
thanks for fixing it!
ideally this configuration should point to localhost and be replaced by config vars on openshift environments.

Reply:
Question: should this point to the "default" or the dev env config?

How do we look at the `invenio.cfg` that is committed? Is it the dev env config generally or the "default"?

Reply:
if there is an env related variable then we commit the local or default in invenio.cfg and in our openshift templates we define the value for each environment respectively

Reply:
it should point to localhost, like the other domain related variables, and in openshift we will overwrite with config
Could you change it to 127.0.0.1:5000 ? it will be ready to merge :)